qoffer-infrastructure/src/main/groovy/life/qbic/portal/qoffer2/customers/CustomerDbConnector.groovy

```diff
@@ -258,28 +261,32 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
     }
   }
 
-  //Fixme no use of closeable
-  private static int getAffiliationId(Connection connection, Affiliation affiliation) {
+  @Override
+  int getAffiliationId(Affiliation affiliation) {
```

Comment:
Why is this method public? The ID is a detail of the implementation and should not be part of the public API.